packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -226,55 +213,66 @@ abstract contract MajorityVotingBase is
 
         return
             _calculatePct(
-                proposal_.yes + proposal_.no + proposal_.abstain,
-                proposal_.totalVotingPower
+                proposal_.tally.yes + proposal_.tally.no + proposal_.tally.abstain,
+                proposal_.tally.totalVotingPower
             );
     }
 
+    /// @inheritdoc IMajorityVoting
+    function supportThreshold() public view returns (uint64) {
```

Comment:
Let's make all these 5 functions virtual..

Reply:
Amended in 5ecb722fe9219deb2ee0fa59d3622c3eddd7418e